src/lib/InputLabel.svelte

```diff
@@ -0,0 +1,8 @@
+<script lang="ts">
+	export let forName: string;
```

Comment:
We can do the same trick we do for `class` here to keep this as `for`

```suggestion
	let forName: string;
	export { forName as for };
```